### PR TITLE
MGMT-9300: Makefile should be consistent on using podman/docker

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
Currently we have a variable to configure which tool (podman/docker)
to use during image build/push. This PR also add a configuration
option for bundle-build/push commands

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -192,11 +192,11 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	$(BUNDLE_CONTAINER_COMMAND) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 .PHONY: bundle-push
 bundle-push: ## Push the bundle image.
-	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
+	$(BUNDLE_CONTAINER_COMMAND) IMG=$(BUNDLE_IMG)
```
Comment (qbarrand) [inline Makefile:199]:
```suggestion
	$(BUNDLE_CONTAINER_COMMAND) push IMG=$(BUNDLE_IMG)
```

Comment (yevgeny-shnaidman) [reply Makefile]: thanks! Fixed

**Pull request merged**